### PR TITLE
Deprecate `limx` and `limy` in `coord_trans()`

### PR DESCRIPTION
**Pull request by billdenney**
The `limx` and `limy` arguments to `coord_trans()` don't align with most other commands.  I've suggested deprecating them and using the more standard `xlim` and `ylim`.

### CODE REVIEW

File: R/coord-transform.r [hunk 1]
```diff
@@ -89,20 +89,27 @@ coord_trans <- function(x = "identity", y = "identity", limx = NULL, limy = NULL
     gg_dep("1.0.1", "`ytrans` arguments is deprecated; please use `y` instead.")
     y <- ytrans
   }
+  if (!missing(limx)) {
+    gg_dep("3.1.0", "`limx` arguments is deprecated; please use `xlim` instead.")
+    if (!is.null(xlim)) {
+      stop("Must not provie both `xlim` and `limx`; please use `xlim` only.")
```
Comment (clauswilke) [inline R/coord-transform.r]: Note typo: "provie" (appears twice)

Comment (billdenney) [reply R/coord-transform.r]: Whoops!  (Fixed)

File: R/coord-transform.r [hunk 2]
```diff
@@ -89,20 +89,27 @@ coord_trans <- function(x = "identity", y = "identity", limx = NULL, limy = NULL
     gg_dep("1.0.1", "`ytrans` arguments is deprecated; please use `y` instead.")
     y <- ytrans
   }
+  if (!missing(limx)) {
+    gg_dep("3.1.0", "`limx` arguments is deprecated; please use `xlim` instead.")
+    if (!is.null(xlim)) {
+      stop("Must not provide both `xlim` and `limx`; please use `xlim` only.")
```
Comment (thomasp85) [inline R/coord-transform.r]: Can you rephrase this to a full sentence? (also the one below)

Comment (billdenney) [reply R/coord-transform.r]: I fixed them and corrected the grammar in the `gg_dep()` warnings.
